cmd/jaeger-clickhouse/main.go

```diff
@@ -6,6 +6,9 @@ import (
 	"os"
 	"path/filepath"
 
+	// Package contains time zone info for connecting to ClickHouse servers with non-UTC time zone
+	_ "time/tzdata"
```

Comment:
> // Package tzdata provides an embedded copy of the timezone database.
// If this package is imported anywhere in the program, then if
// the time package cannot find tzdata files on the system,
// it will use this embedded information.

Reply:
To connect to ClickHouse server with non-UTC time zone, you need to have information about this time zone. Importing this library is universal way to provide such information on every system.